packages/react-server/core/components/FragmentDataCache.jsx

```diff
@@ -1,6 +1,8 @@
 
 var ReactServerAgent = require("../ReactServerAgent"),
-	React = require("react");
+	React = require('react'),
+	PropTypes = require('prop-types'),
+	createReactClass = require("create-react-class");
```

Comment:
Could we avoid this new dependency by making `FragmentDataCache` an es6 class?

Reply:
Sure I'll do that.